tests/unit/misc_test.c

```diff
@@ -183,6 +185,22 @@ CTEST2(misc, test_ctest_assert_prints_user_msg_with_params)
    platform_close_log_stream(&stream, data->log_output);
 }
 
+/*
+ * Test case to exercise platform_error_log() w/ and w/o arguments to msg.
+ */
+CTEST2(misc, test_platform_error_log_wo_args)
+{
+   platform_error_log("This is a test message without arguments\n");
+}
+
+CTEST2(misc, test_platform_error_log_w_args)
+{
+   platform_error_log("This is a test message with arguments: "
```

Comment:
Couple of simple unit tests to show that this new printing stuff will work correctly.

Here is the output using `VERBOSE=1` when running these new test cases:

```
sdb-fdb-build:[41] $ VERBOSE=1 build/release/bin/unit/misc_test test_platform_error_log
Running 1 CTests, suite name 'misc', test case 'test_platform_error_log'.
TEST 1/2 misc:test_platform_error_log_wo_args
Verbose mode on.  This test exercises error-reporting logic, so on success it will print a message that appears to be an error.
tests/unit/misc_test.c:193:ctest_misc_test_platform_error_log_wo_args_run(): OS-pid=3406407, OS-tid=3406407, Thread-ID=0, This is a test message without arguments
[OK]
TEST 2/2 misc:test_platform_error_log_w_args
Verbose mode on.  This test exercises error-reporting logic, so on success it will print a message that appears to be an error.
tests/unit/misc_test.c:198:ctest_misc_test_platform_error_log_w_args_run(): OS-pid=3406407, OS-tid=3406407, Thread-ID=0, This is a test message with arguments: sizeof(pid_t)=4, 'String arg to test msg.'
[OK]
RESULTS: 2 tests (2 ok, 0 failed, 0 skipped) ran in 0 ms
```